src/bin/inx-chronicle/cli.rs

```diff
@@ -82,6 +82,10 @@ pub struct InfluxDbArgs {
     #[cfg(feature = "analytics")]
     #[arg(long, default_value_t = !influxdb::DEFAULT_ANALYTICS_ENABLED)]
     pub disable_analytics: bool,
+    /// Select a subset of analytics to compute. If unset, all analytics will be computed.
+    #[cfg(feature = "analytics")]
+    #[arg(long, value_name = "ANALYTICS")]
+    selected_analytics: Vec<chronicle::db::influxdb::AnalyticsChoice>,
```

Comment:
The flags should be the same here as in the `fill-analytics` subcommand.

```suggestion
    analytics: Vec<chronicle::db::influxdb::AnalyticsChoice>,
```